platform/event/event_payload.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "state_signature_payload.proto";
+
+option java_package = "com.swirlds.proto.legacy.event";
+// <<<pbj.java_package = "com.swirlds.proto.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * The payload of an event, which is either generated by the application or the platform
+ */
+message EventPayload {
+  /**
+   * the payload
+   */
+  oneof payload {
+      /**
+       * application payload, the encoding of the payload is defined by the application
```

Comment:
Field documentation is markdown, except that line breaks are removed and there must not be any blank lines (the processor breaks on those).
HTML tags should be used for block quotes, lists, paragraph breaks and line breaks.
markdown still works for style (e.g. `_italic_`, `**bold**`, etc...)
```suggestion
       * An application payload.<br/>
       * The contents of this payload SHALL be defined by the application or platform
       * subsystem that created the event.<br/>
       * The contents SHOULD be a serialized protobuf message.
```